libraries/AP_DDS/Idl/ardupilot_msgs/msg/Status.idl

```diff
@@ -0,0 +1,52 @@
+// generated from rosidl_adapter/resource/msg.idl.em
+// with input from ardupilot_msgs/msg/Status.msg
+// generated code does not contain a copyright notice
+
+#include "std_msgs/msg/Header.idl"
+
+module ardupilot_msgs {
+  module msg {
+    module Status_Constants {
+      const uint8 APM_ROVER = 1;
+      const uint8 APM_ARDUCOPTER = 2;
+      const uint8 APM_ARDUPLANE = 3;
+      const uint8 APM_ANTENNATRACKER = 4;
+      const uint8 APM_UNKNOWN = 5;
+      const uint8 APM_REPLAY = 6;
+      const uint8 APM_ARDUSUB = 7;
+      const uint8 APM_IOFIRMWARE = 8;
+      const uint8 APM_AP_PERIPH = 9;
+      const uint8 APM_AP_DAL_STANDALONE = 10;
+      const uint8 APM_AP_BOOTLOADER = 11;
+      const uint8 APM_BLIMP = 12;
+      const uint8 APM_HELI = 13;
+      const uint8 FS_RADIO = 1;
```

Comment:
should this be a separate enum? seems odd to fix FS and APM_xxx

Reply:
this is auto generated, I could set the value to start from 14.

Reply:
I set the FS enums starting from 21